_includes/report.html

```diff
@@ -21,7 +21,7 @@ <h2>{{ site.data['locales'][page['lang']]['sections']['translation_report'] }} -
         {% for lesson in section[1] %}
         <tr>
           <td>{{ site.data['locales'][page['lang']]['sections'][title] }}</td>
-          <td><a href ="/en/lessons/{{title}}/{{lesson[1]['chapter']}}">{{ lesson[1]['lesson'] }}</a></td>
+          <td><a href ="{{lesson[1]['url']}}">{{ lesson[1]['lesson'] }}</a></td>
```

Comment:
Thanks, @brain-geek! I really like that approach. Don't you mind if I just cherry-pick your commit into #1455?